lib/cron_parser.dart

```diff
@@ -7,6 +7,7 @@ import 'package:timezone/timezone.dart';
 
 abstract class HasNext<E> {
```

Comment:
@rbubke `HasNext` isn't the name anymore, because there are `next()` **and** `previous()`. Do you have a better name?

Reply:
Second interface HasPrevious  and extend from both?

Reply:
CronIterator?

Reply:
Like this?
```dart
abstract class HasNext<E> {
  /// Find next suitable date
  E next();
}

abstract class HasPrevious<E> {
  /// Find previous suitable date
  E previous();
}

mixin CronIterator<E> on HasPrevious, HasNext {
  E current();
}
```

Reply:
looks good

Reply:
Added with 4ccfbcb4a807090ee472db6f318c8752e9a47cde ✅